tests/extensions/ddl.sql

```diff
@@ -0,0 +1,15 @@
+---
+--- pgcopydb test/extensions/ddl.sql
+---
+--- This file implements DDL changes in the test database.
+
+begin;
+-- Create new tables with foreign key relationships to test
+-- the query for `schema_list_extensions()`
+create table one (id int primary key, two_id int);
+create table two (id int primary key, one_id int, three_id int);
+create table three (id int primary key);
+alter table one add constraint one_two_fk foreign key (two_id) references two(id);
```

Comment:
I left the FK's in their own lines, ordering makes this harder to do all in one because of the loop.

Reply:
I see, makes sense!